src/bokeh/models/plots.py

```diff
@@ -754,25 +754,45 @@ def _check_bad_extra_range_name(self) -> str | None:
     """)
 
     lod_factor = Int(10, help="""
-    Decimation factor to use when applying level-of-detail decimation.
+    Decimation factor to use when applying level-of-detail mode.
+
+    A ``lod_factor`` of N means that only every Nth point in the data source
+    will be drawn while interactive events are active. For example, if
+    ``lod_factor=200`` then only every 200th point will be drawn.
+
+    The level-of-detail mode is intended to preserve interactive response
+    times on HTML canvas plots when there are a large number of data points.
+
+    Note that a possible alternative to level-of-detail mode is using the
+    WebGL ``output_backend``. WebGL rendering may allow very large data sets
+    to remain interactive without any level-of-detail downsampling.
```

Comment:
It would be good to explain that `"webgl"` backend completely disables LOD, either here or in `lod_threshold` (i.e. `output_backend == "webgl" => lod_threshold == None`).

Reply:
[32e1d60](https://github.com/bokeh/bokeh/pull/14368/commits/32e1d60ac4b4ae0ad37ba8137d0a82a77f1653f5)